pkgs/build-support/fetchurl/mirrors.nix

```diff
@@ -151,9 +151,11 @@ rec {
 
   # ImageMagick mirrors, see http://www.imagemagick.org/script/download.php.
   imagemagick = [
+    http://ftp.surfnet.nl/pub/ImageMagick/
+    http://www.imagemagick.org/download/
     ftp://ftp.nluug.nl/pub/ImageMagick/
-    ftp://ftp.imagemagick.org/pub/ImageMagick/
-    ftp://ftp.imagemagick.net/pub/ImageMagick/
+    ftp://ftp.imagemagick.org/pub/ImageMagick/releases/
+    ftp://ftp.imagemagick.net/pub/ImageMagick/releases/
     ftp://ftp.sunet.se/pub/multimedia/graphics/ImageMagick/
   ];
```

Comment:
I tested the mirrors with curl. The first one works for me. The second one word without releases but now doesn't as a new version has come out since my patch. Unfortunately the delete the "-x" releases. In some cases, like here, they've left the "-4" version now that they're at "-5" but they've deleted the prior ones and I'd expect them to delete "-4". I don't know a good workaround for this.